pages/manage/productions/create.vue

```diff
@@ -28,44 +28,14 @@
           <label for="production-name">{{
             $t('productions.production_name')
           }}</label>
-          <b-input
-            id="production-name"
+          <vue-typeahead-bootstrap
```

Comment:
can you give a selected color that matches the udb colors? Instead of the default blue, the $udb-blue for example

Reply:
<img width="870" alt="Screenshot 2020-08-27 at 12 33 03" src="https://user-images.githubusercontent.com/89046/91432060-7cb61180-e861-11ea-98ea-63e20166dd1a.png">

it's not blue, it's light grey. I think it's neutral enough to work here. (screenshot is the selected state)

Reply:
I agree that it looks fine and works, but I just wanted to add that the current autocomplete fields in the AngularJS app look like this:

![Screenshot 2020-08-27 at 13 45 41](https://user-images.githubusercontent.com/959026/91438424-a07e5500-e86b-11ea-8f32-718ec983e59f.png)

I don't think this is necessarily that much better, but it might be helpful to already replicate that look so that when we add more and more Vue pages or re-build ones from the AngularJS app, we don't have two types of autocomplete fields across the app.